.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ orbs:
 executors:
   golang:
     docker:
-      - image: circleci/golang:1.13
+      - image: circleci/golang:1.16.4
```

Comment:
FYI this version is significantly out of date in terms of bugfix releases, the latest is 1.16.15. 1.16.x is also unsupported now that 1.18 is out, but I imagine you're already aware of that.